app/javascript/src/Users/permissions.jsx

```diff
@@ -0,0 +1,278 @@
+/** @jsx element */
```

Comment:
Can you delete this file from this PR ?
It should focus only in updating babel / webpack, not introducing new components

Reply:
We refactored this file weeks ago 😄 . It must've been added by accident during the last rebase.